CODE-OF-CONDUCT.md

```diff
@@ -0,0 +1,35 @@
+# Contributer Code of Conduct
+
+## Sovereign Cloud Stack Community
+
+Sovereign Cloud Stack (SCS) is a network of existing and future providers of standardized sovereign cloud and container infrastructure to join forces in defining, implementing and operating a fully open, federated, compatible platform. We are committed to ["The Four Opens"](https://www.openstack.org/four-opens/) of the [Open Infrastructure Foundation](https://openinfra.dev/) and align our actions according to these four non-negotiable core principles of open collaboration. We do believe that an open and welcoming environment is essential for an active and engaged community. As contributors and maintainers we pledge to making participation in our project and our community a harassment-free experience for everyone, regardless of age, body size, disability, ethnicity, sex characteristics, gender identity and expression, level of experience, education, socio-economic status, nationality, personal appearance, race, religion, or sexual identity and orientation.
+
+No list is ever exhaustive, so we encourage members of the SCS community to adhere to the spirit, rather than the letter, of this code, as that is how it will be enforced. Places where this code may be particularly applicable are GitHub issues and pull requests, Matrix chat, mailing lists, discussions on social networks broadly directed at or between members of the community, and other direct interactions within the community. Any violations, especially continued or flagrant offenses, may affect an individual’s (or organization’s) ability to participate within the SCS community.
+
+We encourage you to confront someone or a group of people that you observe to be in violation of the CoC (whether in letter or in spirit) with your observation, so that the person(s) can correct his/her/their behavior. If this is not successful or it is difficult for you to confront the people and you seek support for this, you may contact the Code of Conduct Committee at [conduct@lists.scs.community](mailto:conduct@lists.scs.community). If you have questions, concerns, or any other inquiries please feel free to contact the project lead at [project@scs.sovereignit.de](mailto:project@scs.sovereignit.de).
```

Comment:
Who is the CoC Committee? How are the members of the CoC Committee selected? 

What if a CoC Committee member and/or project lead is involved in the negative interaction?

Reply:
We have added two lines addressing your questions. The project lead has been removed completely from this document (as it should not be part of the process).